### PR TITLE
Adding DNAC log levels for Discovery and PnP

### PR DESCRIPTION
**Pull request by abimishr**
Problem Description: Logging based on various levels (CRITICAL, ERROR, WARNING, INFO, DEBUG)
Fix: We have added dnac_log_levels
Test cases: Have tested for default (WARNING and DEBUG)
Results: 1. WARNING gives logs for WARNINGS, CRITICAL AND ERROR
              2.  DEBUG logs all the messages including CRITICAL, ERROR, WARNING and INFO as well.

### CODE REVIEW

File: plugins/modules/discovery_intent.py [hunk 1]
```diff
@@ -721,7 +736,7 @@ def get_discovery_device_info(self, discovery_id=None, task_id=None):
             )
             devices = response.response
 
-            self.log(devices)
+            self.log("Devices Information retrieved from the get discovered network devices by discovery id API is {0}".format(str(devices)), "DEBUG")
```
Comment (madhansansel) [inline plugins/modules/discovery_intent.py:739]: self.log("Retrieved device details using the API 'get_discovered_network_devices_by_discovery_id': {0}".format(str(devices)), "DEBUG")

File: plugins/modules/discovery_intent.py [hunk 2]
```diff
@@ -734,8 +749,10 @@ def get_discovery_device_info(self, discovery_id=None, task_id=None):
 
         if not result:
             msg = 'Discovery network device with id {0} has not completed'.format(discovery_id)
+            self.log(msg, "CRITICAL")
             self.module.fail_json(msg=msg)
 
+        self.log('Discovery network device with id {0} has not completed'.format(discovery_id), "INFO")
```
Comment (madhansansel) [inline plugins/modules/discovery_intent.py:755]: if not result, we set the msg with same message (line num: 751) and if the result is true then also we see the same message. Can you please check it?

File: plugins/modules/discovery_intent.py [hunk 3]
```diff
@@ -777,8 +794,9 @@ def delete_exist_discovery(self, params):
             params=params,
         )
 
-        self.log(response)
+        self.log("Response collected from delete discovery by id API is {0}".format(str(response)), "DEBUG")
```
Comment (madhansansel) [inline plugins/modules/discovery_intent.py:797]: self.log("Response collected from API 'delete_discovery_by_id': {0}".format(str(response)), "DEBUG")

File: plugins/modules/pnp_intent.py [hunk 4]
```diff
@@ -318,14 +323,16 @@ def get_site_details(self):
                 params={"name": self.want.get("site_name")},
             )
         except Exception:
+            self.log("Site {0} not found".format(self.want.get("site_name")), "CRITICAL")
             self.module.fail_json(msg="Site not found", response=[])
 
         if response:
-            self.log(str(response))
+            self.log("Site details of {0} received are {1}".format(self.want.get("site_name"), str(response)), "DEBUG")
```
Comment (madhansansel) [inline plugins/modules/pnp_intent.py:330]: self.log("Received site details for '{0}': {1}".format(self.want.get("site_name"), str(response)), "DEBUG")

File: plugins/modules/pnp_intent.py [hunk 5]
```diff
@@ -318,14 +323,16 @@ def get_site_details(self):
...
             site = response.get("response")
             if len(site) == 1:
                 site_id = site[0].get("id")
                 site_exists = True
+                self.log("Site Id is {0} and site name is {1}".format(site_id, self.want.get("site_name")), "INFO")
```
Comment (madhansansel) [inline plugins/modules/pnp_intent.py:335]: Can we say 
                self.log("Site Name: {1},  Site ID: {0}".format(site_id, self.want.get("site_name")), "INFO")

File: plugins/modules/pnp_intent.py [hunk 6]
```diff
@@ -352,15 +359,17 @@ def get_site_type(self):
                 params={"name": self.want.get("site_name")},
             )
         except Exception:
+            self.log("Site {0} not found".format(self.want.get("site_name")), "CRITICAL")
```
Comment (madhansansel) [inline plugins/modules/pnp_intent.py:362]: self.log("Exception occurred as site '{0}' was not found".format(self.want.get("site_name")), "CRITICAL")

File: plugins/modules/pnp_intent.py [hunk 7]
```diff
@@ -352,15 +359,17 @@ def get_site_type(self):
...
             self.module.fail_json(msg="Site not found", response=[])
 
         if response:
-            self.log(str(response))
+            self.log("Site details of {0} received are {1}".format(self.want.get("site_name"), str(response)), "DEBUG")
```
Comment (madhansansel) [inline plugins/modules/pnp_intent.py:366]: self.log("Received site details for '{0}': {1}".format(self.want.get("site_name"), str(response)), "DEBUG")

File: plugins/modules/pnp_intent.py [hunk 8]
```diff
@@ -352,15 +359,17 @@ def get_site_type(self):
...
             site = response.get("response")
             site_additional_info = site[0].get("additionalInfo")
             for item in site_additional_info:
                 if item["nameSpace"] == "Location":
                     site_type = item.get("attributes").get("type")
+                    self.log("Site Type is {0}".format(site_type), "INFO")
```
Comment (madhansansel) [inline plugins/modules/pnp_intent.py:372]: self.log("Site type for site name '{1}' : {0}".format(site_type, site_name), "INFO")

File: plugins/modules/pnp_intent.py [hunk 9]
```diff
@@ -545,9 +559,10 @@ def get_have(self):
                 function='get_device_list',
                 params={"serial_number": self.want.get("serial_number")}
             )
-            self.log(str(device_response))
+            self.log("Device details of the device with serial number {0} are {1}".format(self.want.get("serial_number"), str(device_response)), "DEBUG")
```
Comment (madhansansel) [inline plugins/modules/pnp_intent.py:562]: self.log("Device details for the device with serial number '{0}': {1}".format(self.want.get("serial_number"), str(device_response)), "DEBUG")

File: plugins/modules/pnp_intent.py [hunk 10]
```diff
@@ -517,6 +530,7 @@ def get_reset_params(self):
             ]
         }
 
+        self.log("Paramters used for resetting from deleted state are {0}".format(str(reset_params)), "INFO")
```
Comment (madhansansel) [inline plugins/modules/pnp_intent.py:533]: self.log("Parameters used for resetting from the deleted state: {0}".format(str(reset_params)), "INFO")

File: plugins/modules/pnp_intent.py [hunk 11]
```diff
@@ -566,29 +581,31 @@ def get_have(self):
                     params=self.want.get("image_params"),
                 )
                 image_list = image_response.get("response")
-                self.log(str(image_response))
+                self.log("Image details obtained from the API 'get software image details' are {0}".format(str(image_response)), "DEBUG")
```
Comment (madhansansel) [inline plugins/modules/pnp_intent.py:584]: We can specify the API name directly as the APIs are opensource. 
self.log("Image details obtained from the API 'get_software_image_details': {0}".format(str(image_response)), "DEBUG")

File: plugins/modules/pnp_intent.py [hunk 12]
```diff
@@ -566,29 +581,31 @@ def get_have(self):
...
 
                 # check if project has templates or not
                 template_list = self.dnac_apply['exec'](
                     family="configuration_templates",
                     function='gets_the_templates_available',
                     params={"project_names": self.want.get("project_name")},
                 )
-                self.log(str(template_list))
+                self.log("Templates list under the project {0} is {1}".format(self.want.get("project_name"), str(template_list)), "DEBUG")
```
Comment (madhansansel) [inline plugins/modules/pnp_intent.py:592]: self.log("List of templates under the project '{0}': {1}".format(self.want.get("project_name"), str(template_list)), "DEBUG")

File: plugins/modules/pnp_intent.py [hunk 13]
```diff
@@ -566,29 +581,31 @@ def get_have(self):
...
 
                 dev_details_response = self.dnac_apply['exec'](
                     family="device_onboarding_pnp",
                     function="get_device_by_id",
                     params={"id": device_response[0].get("id")}
                 )
-
+                self.log("Device details of the existing device after calling the API 'get device by id' are {0}".format(str(dev_details_response)), "DEBUG")
```
Comment (madhansansel) [inline plugins/modules/pnp_intent.py:599]: self.log("Device details retrieved after calling the 'get_device_by_id' API: {0}".format(str(dev_details_response)), "DEBUG")

File: plugins/modules/pnp_intent.py [hunk 14]
```diff
@@ -566,29 +581,31 @@ def get_have(self):
...
                 install_mode = dev_details_response.get("deviceInfo").get("mode")
+                self.log("Installation mode of the device with the serial no. {0} is {1}".format(self.want.get("serial_number"), install_mode), "INFO")
```
Comment (madhansansel) [inline plugins/modules/pnp_intent.py:601]: self.log("Installation mode of the device with serial number '{0}': {1}".format(self.want.get("serial_number"), install_mode), "INFO")

File: plugins/modules/pnp_intent.py [hunk 15]
```diff
@@ -566,29 +581,31 @@ def get_have(self):
...
 
                 # check if given site exits, if exists store current site info
                 site_exists = False
                 if not isinstance(self.want.get("site_name"), str) and \
                         not self.want.get('pnp_params')[0].get('deviceInfo'):
                     self.msg = "Name of the site must be a string"
```
Comment (madhansansel) [inline plugins/modules/pnp_intent.py:607]: We can make it simple 
"The site name must be a string."

File: plugins/modules/pnp_intent.py [hunk 16]
```diff
@@ -597,12 +614,13 @@ def get_have(self):
 
                 if site_exists:
                     have["site_id"] = site_id
-                    self.log("Site Exists: " + str(site_exists) + "\n Site_id:" + str(site_id))
-                    self.log("Site Name:" + str(site_name))
+                    self.log("Site Exists: " + str(site_exists) + "\n Site_id:" + str(site_id), "INFO")
```
Comment (madhansansel) [inline plugins/modules/pnp_intent.py:617]: self.log("Site Exists: {0}\nSite Name: {1}\nSite ID: {2}".format(site_exists, site_name, site_id), "INFO")

File: plugins/modules/pnp_intent.py [hunk 17]
```diff
@@ -597,12 +614,13 @@ def get_have(self):
...
+                    self.log("Site Name:" + str(site_name), "INFO")
```
Comment (madhansansel) [inline plugins/modules/pnp_intent.py:618]: Not required, we can club it in the before log message.

File: plugins/modules/pnp_intent.py [hunk 18]
```diff
@@ -597,12 +614,13 @@ def get_have(self):
...
                     if self.want.get("pnp_type") == "access_point":
                         if self.get_site_type() != "floor":
                             self.msg = "Type of the site must \
```
Comment (madhansansel) [inline plugins/modules/pnp_intent.py:621]: self.msg = "The site type must be specified as 'floor' for claiming an AP."

File: plugins/modules/pnp_intent.py [hunk 19]
```diff
@@ -611,36 +629,41 @@ def get_have(self):
                             self.msg = "Installation mode must be in \
                             INSTALL mode to upgrade the image. Current mode is\
                             {0}".format(install_mode)
+                            self.log(str(self.msg), "CRITICAL")
                             self.status = "failed"
                             return self
 
                         have["image_id"] = image_list[0].get("imageUuid")
-                        self.log("Image Id: " + str(have["image_id"]))
+                        self.log("Image Id is {0}".format(str(have["image_id"])), "INFO")
```
Comment (madhansansel) [inline plugins/modules/pnp_intent.py:637]: print image name as well.
                        self.log("Image ID for the image '{0}':  {1}".format(image_name, str(have["image_id"])), "INFO")

File: plugins/modules/pnp_intent.py [hunk 20]
```diff
@@ -611,36 +629,41 @@ def get_have(self):
...
 
                     template_name = self.want.get("template_name")
                     if template_name:
                         if not (template_list and isinstance(template_list, list)):
                             self.msg = "Project Not Found \
```
Comment (madhansansel) [inline plugins/modules/pnp_intent.py:642]: self.msg = "Either project is not found or it is empty."

File: plugins/modules/pnp_intent.py [hunk 21]
```diff
@@ -611,36 +629,41 @@ def get_have(self):
...
+                            self.log(self.msg, "CRITICAL")
                             self.status = "failed"
                             return self
 
                         template_details = get_dict_result(template_list, 'name', template_name)
                         if template_details:
                             have["template_id"] = template_details.get("templateId")
                         else:
-                            self.msg = "Template Not found"
+                            self.msg = "Template {0} Not found".format(template_name)
```
Comment (madhansansel) [inline plugins/modules/pnp_intent.py:652]: self.msg = "Template '{0}' is not found.".format(template_name)

File: plugins/modules/pnp_intent.py [hunk 22]
```diff
@@ -728,13 +753,16 @@ class instance for further use.
                     function='get_device_list',
                     params={"serial_number": device["deviceInfo"]["serialNumber"]}
                 )
-
+                self.log("Device details for the serial number {0}\
```
Comment (madhansansel) [inline plugins/modules/pnp_intent.py:756]: self.log("Device details for serial number {0} obtained from the API 'get_device_list': {1}".format(device["deviceInfo"]["serialNumber"], str(multi_device_response)), "DEBUG")

File: plugins/modules/pnp_intent.py [hunk 23]
```diff
@@ -728,13 +753,16 @@ class instance for further use.
...
+                    obtained from the API 'get device list'\
+                        are {1}".format(device["deviceInfo"]["serialNumber"], str(multi_device_response)), "DEBUG")
                 if (multi_device_response and (len(multi_device_response) == 1)):
                     devices_added.append(device)
-
+                    self.log("Details of the added device is {0}".format(str(device)), "INFO")
```
Comment (madhansansel) [inline plugins/modules/pnp_intent.py:761]: self.log("Details of the added device: {0}".format(str(device)), "INFO")

File: plugins/modules/pnp_intent.py [hunk 24]
```diff
@@ -748,15 +776,18 @@ class instance for further use.
                 params={"payload": bulk_list},
                 op_modifies=True,
             )
+            self.log("Devices imported response from the API 'import_devices_in_bulk' is {0}".format(bulk_params), "DEBUG")
```
Comment (madhansansel) [inline plugins/modules/pnp_intent.py:779]: self.log("Response from API 'import_devices_in_bulk' for imported devices: {0}".format(bulk_params), "DEBUG")

File: plugins/modules/pnp_intent.py [hunk 25]
```diff
@@ -786,15 +818,17 @@ class instance for further use.
                 )
 
                 self.have["deviceInfo"] = dev_add_response.get("deviceInfo")
-                self.log(str(dev_add_response))
+                self.log("Single device addition response from the API 'add device' is {0}".format(str(dev_add_response)), "DEBUG")
```
Comment (madhansansel) [inline plugins/modules/pnp_intent.py:821]: self.log("Response from API 'add device' for a single device addition: {0}".format(str(dev_add_response)), "DEBUG")

File: plugins/modules/pnp_intent.py [hunk 26]
```diff
@@ -808,7 +842,7 @@ class instance for further use.
                     op_modifies=True,
                 )
                 self.have["deviceInfo"] = dev_add_response.get("deviceInfo")
-                self.log(str(dev_add_response))
+                self.log("Single device addition response from the API 'add device'is {0}".format(str(dev_add_response)), "DEBUG")
```
Comment (madhansansel) [inline plugins/modules/pnp_intent.py:845]: self.log("Response from API 'add device' for single device addition: {0}".format(str(dev_add_response)), "DEBUG")

File: plugins/modules/pnp_intent.py [hunk 27]
```diff
@@ -818,16 +852,18 @@ class instance for further use.
                     params=claim_params,
                 )
 
-                self.log(str(claim_response))
+                self.log("Single claiming response from the API 'claim a device to a site' is {0}".format(str(dev_add_response)), "DEBUG")
```
Comment (madhansansel) [inline plugins/modules/pnp_intent.py:855]: self.log("Response from API 'claim a device to a site' for a single claiming: {0}".format(str(dev_add_response)), "DEBUG")

File: plugins/modules/pnp_intent.py [hunk 28]
```diff
@@ -838,23 +874,30 @@ class instance for further use.
             op_modifies=True,
             params=provisioned_count_params,
         )
+        self.log("Devices which are provisioned response from the API 'get device count' is {0}".format(str(prov_dev_response)), "DEBUG")
```
Comment (madhansansel) [inline plugins/modules/pnp_intent.py:877]: self.log("Response from 'get device count' API for provisioned devices: {0}".format(str(prov_dev_response)), "DEBUG")

File: plugins/modules/pnp_intent.py [hunk 29]
```diff
@@ -838,23 +874,30 @@ class instance for further use.
...
+
         plan_dev_response = self.dnac_apply['exec'](
             family="device_onboarding_pnp",
             function='get_device_count',
             op_modifies=True,
             params=planned_count_params,
         )
+        self.log("Devices which are in planned state response from the API 'get device count' is {0}".format(str(plan_dev_response)), "DEBUG")
```
Comment (madhansansel) [inline plugins/modules/pnp_intent.py:885]: self.log("Response from 'get_device_count' API for devices in planned state: {0}".format(str(plan_dev_response)), "DEBUG")

File: plugins/modules/pnp_intent.py [hunk 30]
```diff
@@ -838,23 +874,30 @@ class instance for further use.
...
+
         dev_details_response = self.dnac_apply['exec'](
             family="device_onboarding_pnp",
             function="get_device_by_id",
             params={"id": self.have["device_id"]}
         )
+        self.log("Device's details from the API 'get device by id' is {0}".format(str(dev_details_response)), "DEBUG")
```
Comment (madhansansel) [inline plugins/modules/pnp_intent.py:892]: self.log("Response from 'get_device_by_id' API for device details: {0}".format(str(dev_details_response)), "DEBUG")

File: plugins/modules/pnp_intent.py [hunk 31]
```diff
@@ -838,23 +874,30 @@ class instance for further use.
...
 
         pnp_state = dev_details_response.get("deviceInfo").get("state")
+        self.log("PnP state of the device is {0}".format(pnp_state), "INFO")
```
Comment (madhansansel) [inline plugins/modules/pnp_intent.py:895]: self.log("PnP state of the device: {0}".format(pnp_state), "INFO")

File: plugins/modules/pnp_intent.py [hunk 32]
```diff
@@ -865,7 +908,7 @@ class instance for further use.
                     "payload": update_payload},
             op_modifies=True,
         )
-        self.log(str(update_response))
+        self.log("Update in the device's config API response is {0}".format(str(update_response)), "DEBUG")
```
Comment (madhansansel) [inline plugins/modules/pnp_intent.py:911]: self.log("Response from 'update_device' API for device's config update: {0}".format(str(update_response)), "DEBUG")

File: plugins/modules/pnp_intent.py [hunk 33]
```diff
@@ -875,8 +918,9 @@ class instance for further use.
                 params={"payload": reset_paramters},
                 op_modifies=True,
             )
-            self.log(str(reset_response))
+            self.log("Errored state resolution response from the API is {0}".format(str(reset_response)), "DEBUG")
```
Comment (madhansansel) [inline plugins/modules/pnp_intent.py:921]: self.log("Response from 'update_device' API for errored state resolution: {0}".format(str(reset_response)), "DEBUG")

File: plugins/modules/pnp_intent.py [hunk 34]
```diff
@@ -888,22 +932,24 @@ class instance for further use.
...
             self.result['response'] = self.have.get("device_found")
             self.result['msg'] = "Device is already claimed"
+            self.log(self.result['msg'], "WARNING")
             if update_response.get("deviceInfo"):
                 self.result['changed'] = True
                 return self
 
         claim_params = self.get_claim_params()
-        self.log(str(claim_params))
+        self.log("Paramters for claiming the device are {0}".format(str(claim_params)), "DEBUG")
```
Comment (madhansansel) [inline plugins/modules/pnp_intent.py:941]: self.log("Parameters for claiming the device: {0}".format(str(claim_params)), "DEBUG")

File: plugins/modules/pnp_intent.py [hunk 35]
```diff
@@ -934,7 +980,7 @@ def get_diff_deleted(self):
                 function='get_device_list',
                 params={"serial_number": device["deviceInfo"]["serialNumber"]}
             )
-
+            self.log("Device details for the serial number {0} are {1}".format(device["deviceInfo"]["serialNumber"], str(multi_device_response)), "DEBUG")
```
Comment (madhansansel) [inline plugins/modules/pnp_intent.py:983]: self.log("Response from 'claim_a_device_to_a_site' API for claiming: {0}".format(str(claim_response)), "DEBUG")

File: plugins/modules/pnp_intent.py [hunk 36]
```diff
@@ -945,22 +991,26 @@ def get_diff_deleted(self):
                     params={"id": device_id},
                 )
 
-                self.log(str(response))
+                self.log("Device details for the device deleted with \
```
Comment (madhansansel) [inline plugins/modules/pnp_intent.py:994]: self.log("Device details for the deleted device with serial number '{0}': {1}".format(device["deviceInfo"]["serialNumber"], str(response)), "DEBUG")

File: plugins/modules/discovery_intent.py (hunk 1 above)
Comment (abimishr) [reply plugins/modules/discovery_intent.py:739]: Resolved

File: plugins/modules/discovery_intent.py (hunk 2 above)
Comment (abimishr) [reply plugins/modules/discovery_intent.py:755]: Thanks for pointing it. Resolved

File: plugins/modules/discovery_intent.py (hunk 3 above)
Comment (abimishr) [reply plugins/modules/discovery_intent.py:797]: Resolved

File: plugins/modules/pnp_intent.py (hunk 4 above)
Comment (abimishr) [reply plugins/modules/pnp_intent.py]: Resolved

File: plugins/modules/pnp_intent.py (hunk 5 above)
Comment (abimishr) [reply plugins/modules/pnp_intent.py]: Resolved

File: plugins/modules/pnp_intent.py (hunk 6 above)
Comment (abimishr) [reply plugins/modules/pnp_intent.py]: Resolved

File: plugins/modules/pnp_intent.py (hunk 7 above)
Comment (abimishr) [reply plugins/modules/pnp_intent.py]: Resolved

File: plugins/modules/pnp_intent.py (hunk 8 above)
Comment (abimishr) [reply plugins/modules/pnp_intent.py]: Resolved

File: plugins/modules/pnp_intent.py (hunk 9 above)
Comment (abimishr) [reply plugins/modules/pnp_intent.py]: Resolved

File: plugins/modules/pnp_intent.py (hunk 10 above)
Comment (abimishr) [reply plugins/modules/pnp_intent.py]: Resolved

File: plugins/modules/pnp_intent.py (hunk 11 above)
Comment (abimishr) [reply plugins/modules/pnp_intent.py]: Resolved

File: plugins/modules/pnp_intent.py (hunk 12 above)
Comment (abimishr) [reply plugins/modules/pnp_intent.py]: Resolved

File: plugins/modules/pnp_intent.py (hunk 13 above)
Comment (abimishr) [reply plugins/modules/pnp_intent.py]: Resolved

File: plugins/modules/pnp_intent.py (hunk 14 above)
Comment (abimishr) [reply plugins/modules/pnp_intent.py]: Resolved

File: plugins/modules/pnp_intent.py (hunk 15 above)
Comment (abimishr) [reply plugins/modules/pnp_intent.py]: Resolved (Taken a note)

File: plugins/modules/pnp_intent.py (hunk 16 above)
Comment (abimishr) [reply plugins/modules/pnp_intent.py]: Resolved

File: plugins/modules/pnp_intent.py (hunk 17 above)
Comment (abimishr) [reply plugins/modules/pnp_intent.py]: Done

File: plugins/modules/pnp_intent.py (hunk 18 above)
Comment (abimishr) [reply plugins/modules/pnp_intent.py]: Makes sense

File: plugins/modules/pnp_intent.py (hunk 19 above)
Comment (abimishr) [reply plugins/modules/pnp_intent.py]: Resolved

File: plugins/modules/pnp_intent.py (hunk 20 above)
Comment (abimishr) [reply plugins/modules/pnp_intent.py]: Resolved

File: plugins/modules/pnp_intent.py (hunk 21 above)
Comment (abimishr) [reply plugins/modules/pnp_intent.py]: Resolved

File: plugins/modules/pnp_intent.py (hunk 22 above)
Comment (abimishr) [reply plugins/modules/pnp_intent.py]: Resolved

File: plugins/modules/pnp_intent.py (hunk 23 above)
Comment (abimishr) [reply plugins/modules/pnp_intent.py]: Resolved

File: plugins/modules/pnp_intent.py (hunk 24 above)
Comment (abimishr) [reply plugins/modules/pnp_intent.py]: Resolved

File: plugins/modules/pnp_intent.py (hunk 25 above)
Comment (abimishr) [reply plugins/modules/pnp_intent.py]: Resolved

File: plugins/modules/pnp_intent.py (hunk 26 above)
Comment (abimishr) [reply plugins/modules/pnp_intent.py]: Resolved

File: plugins/modules/pnp_intent.py (hunk 27 above)
Comment (abimishr) [reply plugins/modules/pnp_intent.py]: Resolved

File: plugins/modules/pnp_intent.py (hunk 28 above)
Comment (abimishr) [reply plugins/modules/pnp_intent.py]: Resolved

File: plugins/modules/pnp_intent.py (hunk 29 above)
Comment (abimishr) [reply plugins/modules/pnp_intent.py]: Resolved

File: plugins/modules/pnp_intent.py (hunk 30 above)
Comment (abimishr) [reply plugins/modules/pnp_intent.py]: Resolved

File: plugins/modules/pnp_intent.py (hunk 31 above)
Comment (abimishr) [reply plugins/modules/pnp_intent.py]: Resolved

File: plugins/modules/pnp_intent.py (hunk 32 above)
Comment (abimishr) [reply plugins/modules/pnp_intent.py]: Resolved

File: plugins/modules/pnp_intent.py (hunk 33 above)
Comment (abimishr) [reply plugins/modules/pnp_intent.py]: Resolved

File: plugins/modules/pnp_intent.py (hunk 34 above)
Comment (abimishr) [reply plugins/modules/pnp_intent.py]: Resolved

File: plugins/modules/pnp_intent.py (hunk 35 above)
Comment (abimishr) [reply plugins/modules/pnp_intent.py]: Resolved

File: plugins/modules/pnp_intent.py (hunk 36 above)
Comment (abimishr) [reply plugins/modules/pnp_intent.py]: Resolved

**Review by madhansansel — APPROVED** [review]

**Pull request merged**